### PR TITLE
Fix Reports

### LINKED ISSUE #83: KeyError Triggered Viewing Reports
### Environment
* Python version:  3.8.10
* Nautobot version:  1.2.10
* nautobot-plugin-device-lifecycle-mgmt version:  1.0.2

### Expected Behavior
When you click on reports you should have them displayed.

### Observed Behavior
When you click on the report links you get a Nautobot error screen noting:

```
<class 'KeyError'>
'barchart_bar_width'
Python version: 3.8.10
Nautobot version: 1.2.10
```

### Steps to Reproduce
1. Install Nautobot and DLC plugin.
2. Add DLC data.
3. Click report links.

### PR DESCRIPTION
**Pull request by jdrew82**
This PR should make the required changes to fix #83.

### CODE REVIEW

File: nautobot_device_lifecycle_mgmt/__init__.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class DeviceLifeCycleConfig(PluginConfig):
     author_email = "opensource@networktocode.com"
     description = "Manages device lifecycle of Nautobot Devices and Components."
     base_url = "nautobot-device-lifecycle-mgmt"
-    required_settings = []
+    required_settings = ["barchart_bar_width", "barchart_width", "barchart_height"]
     min_version = "1.0.0"
     max_version = "1.9999"
     default_settings = {"expired_field": "end_of_support"}
```
Comment (jvanderaa) [inline nautobot_device_lifecycle_mgmt/__init__.py:35]: @jdrew82 can we add those defaults here as well?

**Review by jvanderaa — CHANGES_REQUESTED** [review]

**Review by jvanderaa — APPROVED** [review]

**Comment by jvanderaa** [comment]
@progala @josh-silvas what are the thoughts here?

**Comment by progala** [comment]
LGTM. Thanks @jdrew82 !

**Review by progala — APPROVED** [review]
LGTM

**Pull request merged**